django/contrib/admin/options.py

```diff
@@ -962,12 +962,22 @@ def log_deletion(self, request, obj, object_repr):
             action_flag=DELETION,
         )
 
-    @display(description=mark_safe('<input type="checkbox" id="action-toggle">'))
+    @display(
+        description=mark_safe(
+            '<input type="checkbox" id="action-toggle" '
+            'aria-label="Select all objects on this page for an action">'
+        )
+    )
     def action_checkbox(self, obj):
         """
         A list_display column containing a checkbox widget.
         """
-        return helpers.checkbox.render(helpers.ACTION_CHECKBOX_NAME, str(obj.pk))
```

Comment:
`helpers.checkbox` is not unused. I think we can remove it with a small release note:
```
* The undocumented `django.contrib.admin.helpers.checkbox` is removed.
```

Reply:
Does this imply any other changes besides removing the `checkbox` from the `helpers` module? Are the release notes in some django source file?

```python
# /django/django/contrib/admin/helpers.py:39
checkbox = forms.CheckboxInput({"class": "action-select"}, lambda value: False)
```

Reply:
> Does this imply any other changes besides removing the `checkbox` from the `helpers` module?

I don't think so.

> Are the release notes in some django source file?

Yes you should add it to the _"Backwards incompatible changes in 5.0" -> "Miscellaneous"_ section: 

https://github.com/django/django/blob/292aacaf6c3d6956ca2c51c41e36dbf425389346/docs/releases/5.0.txt#L241-L242

Reply:
`checkbox` is still in the `admin.helpers`. Please remove it.